assets/locales/service.cy.toml

```diff
@@ -140,6 +140,10 @@ one = "Data"
 description = "Methodology"
 one = "Methodology"
 
+[YouMightAlsoBeInterestedIn]
+description = "You might also be interested in"
+one = "You might also be interested in"
```

Comment:
Any chance we could get the Welsh translation for this?

Reply:
New ticket will be created for the welsh translations.